apps/guide/content/docs/legacy/interactions/modals.mdx

```diff
@@ -2,30 +2,27 @@
 title: Modals
 ---
 
-With modals you can create pop-up forms that allow users to provide you with formatted inputs through submissions. We'll cover how to create, show, and receive modal forms using discord.js!
+Modals are pop-up forms that allow you to prompt users for additional input. This form-like interaction response blocks the user from interacting with Discord until the modal is submitted or dismissed. In this section we will cover how to create, show, and receive modals using discord.js!
 
 <Callout>
-	This page is a follow-up to the [interactions (slash commands) page](../slash-commands/advanced-creation). Please
-	carefully read that section first, so that you can understand the methods used in this section.
+	This page is a follow-up to the [interactions (slash commands) page](../slash-commands/advanced-creation). Reading
+	that page first will help you understand the concepts introduced in this page.
 </Callout>
 
 ## Building and responding with modals
 
-Unlike message components, modals aren't strictly components themselves. They're a callback structure used to respond to interactions.
+Modals are a way to respond to a user interaction. Submitting the modal in turn will send a new interaction you can respond to. Discord.js offers the `ModalBuilder` class as a convenient way to build modals step by step using setters and callbacks.
```

Comment:
Unsure about the phrasing of "user interaction", since this is somewhat connecting to User Context Menu interactions.